js/index.js

```diff
@@ -0,0 +1,79 @@
+import { Game } from './modules/game.js'; // eslint-disable-line import/extensions
+
+const Player = (id, name, symbol) => ({ id, name, symbol });
```

Comment:
Move it into `game.js`

Reply:
Make the `game.js` file the module with no code related to DOM.
I mean code with only Logic, controller, etc...
And into `index.js` only DOM manipulation in respect to the output of `game.js`